cmd/lotus-shed/sectors.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/ipfs/go-cid"
 	"github.com/urfave/cli/v2"
 	"golang.org/x/xerrors"
+	"gopkg.in/cheggaaa/pb.v1"
```

Comment:
What's this import for?

Reply:
Progress bar

Reply:
Are we ok using this from a licensing perspective? This is under BSD-3

Reply:
seems to be ok based on this post https://opensource.stackexchange.com/questions/10817/can-i-link-a-bsd-3-library-to-an-mit-project#:~:text=The%20BSD%2D3%20library%20will,so%20the%20compile%20can%20pass
 but I'm not a lawyer

Reply:
We're already using it in many other places